projects/kit/utils/mask/create-time-mask.ts

```diff
@@ -1,13 +1,39 @@
 import {TuiTimeMode} from '@taiga-ui/cdk';
 import {TUI_DIGIT_REGEXP, TuiTextMaskList} from '@taiga-ui/core';
-import {TUI_TIME_MASK} from '@taiga-ui/kit/constants';
+import {MAX_TIME_VALUES} from '@taiga-ui/kit/constants';
+import {TuiTimeFormatParts} from '@taiga-ui/kit/types';
 
-export function tuiCreateTimeMask(mode: TuiTimeMode): TuiTextMaskList {
+function tuiCreateTimePartMask(
+    maxPartValue: number | undefined,
+    prefix?: string,
+): Array<string | RegExp> {
+    const partLength =
+        typeof maxPartValue === 'number' ? maxPartValue.toString().length : 0;
```

Comment:
`String(x)` is a slightly shorter way to write `x.toString()` ;)

Reply:
I'm aware of it, thanks for your concern) this form is just a little more explicit